src/thunderbots/software/test/util/ros_messages.cpp

```diff
@@ -255,6 +255,90 @@ TEST(ROSMessageUtilTest, convert_team_with_no_goalie_to_ros_message)
     EXPECT_EQ(4.4, team_msg.robots.at(0).timestamp_seconds);
 }
 
+TEST(ROSMessageUtilTest, transform_ball)
+{
+    Ball ball = Ball(Point(-1.3, 0.2), Vector(-1.3, 5.4), Timestamp::fromSeconds(123.45));
+    thunderbots_msgs::Ball ball_msg = Util::ROSMessages::convertBallToROSMessage(ball);
+
+    thunderbots_msgs::Ball transformed_ball_msg =
+        Util::ROSMessages::transformBallMessage(ball_msg);
+
+    EXPECT_EQ(1.3, transformed_ball_msg.position.x);
+    EXPECT_EQ(-0.2, transformed_ball_msg.position.y);
+    EXPECT_EQ(1.3, transformed_ball_msg.velocity.x);
+    EXPECT_EQ(-5.4, transformed_ball_msg.velocity.y);
+    EXPECT_EQ(Timestamp::fromSeconds(123.45).getSeconds(),
+              transformed_ball_msg.timestamp_seconds);
+}
+
+TEST(ROSMessageUtilTest, transform_robot)
+{
+    Robot robot = Robot(1, Point(1, -5.01), Vector(-2, 3), Angle::quarter(),
+                        AngularVelocity::ofRadians(1), Timestamp::fromSeconds(4.4));
+    thunderbots_msgs::Robot robot_msg =
+        Util::ROSMessages::convertRobotToROSMessage(robot);
+
+    thunderbots_msgs::Robot transformed_robot_msg =
+        Util::ROSMessages::transformRobotMessage(robot_msg);
+
+    EXPECT_EQ(1, transformed_robot_msg.id);
+    EXPECT_EQ(-1, transformed_robot_msg.position.x);
+    EXPECT_EQ(5.01, transformed_robot_msg.position.y);
+    EXPECT_EQ(2, transformed_robot_msg.velocity.x);
+    EXPECT_EQ(-3, transformed_robot_msg.velocity.y);
+    EXPECT_EQ(Angle::quarter().toRadians() + Angle::half().toRadians(),
+              transformed_robot_msg.orientation);
+    EXPECT_EQ(AngularVelocity::ofRadians(1).toRadians(),
+              transformed_robot_msg.angular_velocity);
+    EXPECT_EQ(Timestamp::fromSeconds(4.4).getSeconds(),
+              transformed_robot_msg.timestamp_seconds);
+}
+
```

Comment:
Gareth has a good point on one of my past PRs. I'd like it if we saw a brief comment for each test indicating what it's testing

Reply:
IMO a reasonable alternative to this is really descriptive test names. It's totally fine (and better) to have really long and descriptive test names, since it makes it really obvious what the test case was that failed if it ever shows up in a log, and also describes what's being tested.

That being said, comments are also a good addition for anything reasonably complicated.